### ISSUE TITLE
Option To Filter In-active Players in stats

### ISSUE THREAD

**Issue by zipy124** [Dragaera/observatory#45]
This would be cool so you could see where you are in terms of the active player base. This would make #15 even more useful, as percentile isn't useful unless its of the recent player base not people in 2017.

**Comment by Lavode** [comment]
Agreed, it'd be nice to have something along those lines - be it in the form of a second board which only takes active players into account, or another way.

Realistically I've not been working on this project all too much for many months, so I'd not want people to hope it'll happen anytime soon though, I'm afraid.
